### PR TITLE
C++11 cleanup on constructors guiEngine

### PR DESCRIPTION
**Pull request by Dumbeldor**

### CODE REVIEW

File: src/guiEngine.h [hunk 1]
```diff
@@ -150,12 +150,12 @@ class GUIEngine {
...
+	GUIEngine(irr::IrrlichtDevice *dev,
 			JoystickController *joystick,
-			gui::IGUIElement* parent,
+			gui::IGUIElement *parent,
 			IMenuManager *menumgr,
-			scene::ISceneManager* smgr,
-			MainMenuData* data,
+			scene::ISceneManager *smgr,
+			MainMenuData *data,
 			bool& kill);
```
Comment (SmallJoker) [inline src/guiEngine.h]: If already fixing it: `bool &kill`

File: src/guiEngine.cpp [hunk 2]
```diff
@@ -130,12 +114,12 @@ void MenuMusicFetcher::fetchSounds(const std::string &name,
 /******************************************************************************/
 /** GUIEngine                                                                 */
 /******************************************************************************/
-GUIEngine::GUIEngine(	irr::IrrlichtDevice* dev,
+GUIEngine::GUIEngine(	irr::IrrlichtDevice *dev,
```
Comment (SmallJoker) [inline src/guiEngine.cpp]: Remove the tab after `(`

File: src/guiEngine.cpp [hunk 3]
```diff
@@ -130,12 +114,12 @@ void MenuMusicFetcher::fetchSounds(const std::string &name,
...
 						JoystickController *joystick,
-						gui::IGUIElement* parent,
+						gui::IGUIElement *parent,
 						IMenuManager *menumgr,
-						scene::ISceneManager* smgr,
-						MainMenuData* data,
+						scene::ISceneManager *smgr,
+						MainMenuData *data,
 						bool& kill) :
```
Comment (SmallJoker) [inline src/guiEngine.cpp]: `bool &kill`here too

File: src/guiEngine.h (hunk 1 above)
Comment (SmallJoker) [inline src/guiEngine.h]: `bool &kill`

**Comment by nerzhul** [comment]
@SmallJoker i fixed both cases

**Comment by Dumbeldor** [comment]
Thanks @nerzhul

**Pull request merged**